packages/gafl-webapp-service/src/handlers/__tests__/analytics-handler.spec.js

```diff
@@ -26,28 +26,30 @@ describe('The analytics handler', () => {
       'https://testserver-example-fish',
       '/buy/renew/identify'
     ]
-  ])('redirects to correct page if url matches header', async (origin, referer, host, redirect) => {
+  ])('redirects to correct page if the HTTP_REFERER host matchesthe host of the current page', async (origin, referer, host, redirect) => {
     const payload = {
       analyticsResponse: 'accept'
     }
     const request = generateRequestMock(payload, 'analytics', origin, referer, host)
     const responseToolkit = generateResponseToolkitMock()
+    addLanguageCodeToUri.mockReturnValueOnce(redirect)
```

Comment:
This will make it so your test always passes - no matter what's passed to addLanguageCodeToUri. Outside of this function, at the top of the file, do:
```
jest.mock('../../processors/uri-helper.js', () => ({
  addLanguageCodeToUri: uri => uri
}))
```